Makefile

```diff
@@ -140,6 +126,12 @@ prep-whonix: ## enables apparmor on whonix-ws-14 and whonix-gw-14
 	qvm-prefs -s whonix-gw-14 kernelopts "nopat apparmor=1 security=apparmor"
 	qvm-prefs -s whonix-ws-14 kernelopts "nopat apparmor=1 security=apparmor"
 
+list-vms: ## Prints all Qubes VMs managed by Workstation salt config
+	@./scripts/list-vms
+
+destroy-all: ## Destroys all VMs managed by Workstation salt config
+	@./scripts/list-vms | xargs ./scripts/destroy-vm
```

Comment:
These two targets are far from necessary, but they strike me as super convenient. So problems with removing if they strike you as cruft, @joshuathayer.